pkg/api/norman/customization/monitor/handler_utils.go

```diff
@@ -21,6 +21,7 @@ import (
 
 const (
 	prometheusReqTimeout = 30 * time.Second
+	projectIDAnn         = "field.cattle.io/projectId"
```

Comment:
Why is this added?

Reply:
Because it was coded in the cluster_graph_handler even though it is leveraged by this and the project handler module.  I have removed the cluster graph handler since it's associated CRD no longer exists.